src/core/Effort/service/EffortService.ts

```diff
@@ -138,4 +138,11 @@ export class EffortService {
   async delete(uid: string) {
     await this.effortRepository.delete(uid)
   }
+
+  async deleteEffortByCompany(companyDomain: string) {
+    const allUsersCompany = await this.userProfileService.getByCompany(companyDomain)
```

Comment:
Se dovesse esserci un errore ad un certo punto del ciclo for la cancellazione delle entità "effort" verrebbe interrotta lasciando il DB in stato inconsistente